apps/server/swagger/schemas.yml

```diff
@@ -122,10 +122,25 @@ components:
               type: string
         errors:
           type: object
-          additionalProperties:
-            type: array
-            items:
-              $ref: '#/components/schemas/ValidationError'
+          properties:
+            inserts:
```

Comment:
to clarify for my own understanding of what I'm looking at:
- is this a conjugated verb? (e.g. "the endpoint inserts X")
- or is it a plural? (e.g. "an array of something called inserts").

Reply:
This is the plural form of `insert`, consist on an key-value object with the new data to be inserted grouped by schema name. Similarly this applies for `update` and `delete`. As Submission can consist on insert, update or delete one or more records. 

e.g. 
```
 "data": {
    "inserts": {
      "sample": {
        "records": [
          {
            "study_id": "TEST_PRO",
            "sample_type": "Protein",
            "submitter_sample_id": "TM-01-1",
            "submitter_specimen_id": "LOD-04",
            "submitter_participant_id": "NR-30"
          }
        ],
        "batchName": "sample.tsv"
      }
    },
    "updates": {
      "participant": [
        {
          "new": {
            "date_of_birth": "1990-05"
          },
          "old": {
            "date_of_birth": "1970-05"
          },
          "systemId": "I7R9DH9OP59Y1EZE6PSN9"
        }
      ]
    },
    "deletes": {
      "specimen": [
        {
          "data": {
            "study_id": "TEST_PRO",
            "submitter_specimen_id": "LOD-01",
            "specimen_tissue_source": "Saliva",
            "specimen_collection_date": "2000-01",
            "submitter_participant_id": "NR-01",
            "specimen_anatomic_location": "C33.2"
          },
          "isValid": true,
          "systemId": "28UMRXZHF2XNQTIRQI4AZ",
          "entityName": "specimen",
          "organization": "oicr"
        }
      ]
    }
  },
```

errors are displayed the same way grouped by inserts, updates and deletes:
e.g. 
```
"errors": {
    "inserts": {
      "sample": [
        {
          "index": 0,
          "reason": "INVALID_BY_FOREIGNKEY",
          "fieldName": "submitter_participant_id",
          "fieldValue": "NR-30",
          "foreignSchema": {
            "fieldName": "submitter_participant_id",
            "schemaName": "participant"
          }
        }
      ]
    }
  }
```